x-pack/plugins/uptime_monitoring/public/components/queries/monitor_list/monitor_list.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiHealth,
+  // @ts-ignore missing type definition
+  EuiInMemoryTable,
+  // @ts-ignore missing type definition
+  EuiLineSeries,
+  EuiPanel,
+  // @ts-ignore missing type definition
+  EuiSeriesChart,
+  // @ts-ignore missing type definition
+  EuiSeriesChartUtils,
+  EuiTitle,
+} from '@elastic/eui';
+import moment from 'moment';
+import React, { Fragment } from 'react';
+import { Query } from 'react-apollo';
+import { Link } from 'react-router-dom';
+import { getMonitorListQuery } from './get_monitor_list';
+
+interface MonitorListProps {
+  autorefreshInterval: number;
+  autorefreshEnabled: boolean;
+  dateRangeStart: number;
+  dateRangeEnd: number;
+  filters?: string;
+}
+
+export const MonitorList = ({
+  autorefreshInterval,
+  autorefreshEnabled,
+  dateRangeStart,
+  dateRangeEnd,
+  filters,
+}: MonitorListProps) => (
+  <Query
+    pollInterval={autorefreshEnabled ? autorefreshInterval : undefined}
+    query={getMonitorListQuery}
+    variables={{ dateRangeStart, dateRangeEnd, filters }}
+  >
+    {({ loading, error, data }) => {
+      if (loading) {
+        return 'Loading...';
```

Comment:
Translation needed

Reply:
++ - I wasn't sure if this would be necessary since it's beta/experimental. I'll go through and try to add them as needed, will put a link to the commit diff here when it's done.

Reply:
I'll create a follow-up issue targeted at 6.7.0 for translating the plugin.

Reply:
https://github.com/elastic/kibana/issues/27524

cc @ruflin

Reply:
Let's skip the translation for now.